website/src/documentation/runners/flink.md

```diff
@@ -170,11 +249,70 @@ When executing your pipeline with the Flink Runner, you can set these pipeline o
   <td>Sets the delay between executions. A value of <code>-1</code> indicates that the default value should be used.</td>
   <td><code>-1</code></td>
 </tr>
+<tr>
+  <td><code>objectReuse</code></td>
+  <td>Sets the behavior of reusing objects.</td>
+  <td><code>false</code>, no Object reuse</td>
+</tr>
 <tr>
   <td><code>stateBackend</code></td>
   <td>Sets the state backend to use in streaming mode. The default is to read this setting from the Flink config.</td>
   <td><code>empty</code>, i.e. read from Flink config</td>
 </tr>
+<tr>
```

Comment:
Listing all options here requires ongoing updates to keep in sync with code. Should we refer to the javadoc for FlinkPipelineOptions instead?

Reply:
As a side note, we should develop a checklist for the Flink Runner to ensure documentation gets updated when features are merged. Moving forward this will become more important.

Reply:
It's a lot of detail that some users eventually will need. Can we move the table to the bottom of the page so other sections don't get lost / overlooked?

The options are in Java style, but the portable representation is lower case with underscore (and so is the Python SDK). Should we change that?

I like the idea of generating this, it would be even better if the command line produces this list (we already have it in a different format with the help flag). And we should probably mention that help flag before the table, since that will always provide the latest options.

Reply:
We could use the Java/Python switch to present two versions of the options (one with camel case, one without). Moving to the bottom makes sense.

>I like the idea of generating this, it would be even better if the command line produces this list (we already have it in a different format with the help flag)

Absolutely, that would be nice. `--help` currently only prints the top-level options. I'm assuming it would now print also the retrieved options?

Reply:
Yes, we need to enable Python SDK to output all options. For Java that should already be the case.